_docs-sources/pipelines/using-pipelines/index.md

```diff
@@ -44,13 +44,27 @@ The following is an example of the recommended settings for branch protection:
    - On Success, a comment is placed on the PR indicating success
    - On Failure, a new GitHub issue is created describing the failure. A new PR must be created to resolve any failures.
 
-## GitHub Enterprise Users
+## Allowing Pipelines Actions in GitHub Actions
 
-Gruntwork Pipelines uses a set of Gruntwork built re-usable Github Actions. Companies using GitHub Enterprise may need to explicitly allow Actions from Gruntwork to run in their GitHub organization. See the "Allow specified actions and reusable workflows" section in [Allowing select actions and reusable workflows to run](https://docs.github.com/en/enterprise-cloud@latest/admin/policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-github-actions-in-your-enterprise#allowing-select-actions-and-reusable-workflows-to-run) to learn more.
+Gruntwork Pipelines uses a set of Gruntwork built re-usable Github Actions, which are available in the GitHub Actions marketplace. Gruntwork is a Verified Creator of GitHub Actions.
 
-You will need to allow the following Actions to run:
-- [pipelines-dispatch](https://github.com/gruntwork-io/pipelines-dispatch)
-- [pipelines-orchestrate](https://github.com/gruntwork-io/pipelines-orchestrate)
-- [pipelines-execute](https://github.com/gruntwork-io/pipelines-execute)
+### GitHub Enterprise
 
-Navigate to each repository to retrieve the latest tagged release for each action.
+Gruntwork recommends explicitly allowing Actions required by Gruntwork Pipelines to run in your GitHub organization. See [Allowing select actions and reusable workflows to run](https://docs.github.com/en/enterprise-cloud@latest/admin/policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-github-actions-in-your-enterprise#allowing-select-actions-and-reusable-workflows-to-run) to learn more.
+
+You will need to allow the following GitHub Actions for Gruntwork Pipelines to run successfully:
+- [gruntwork-io/pipelines-dispatch](https://github.com/gruntwork-io/pipelines-dispatch)
+- [gruntwork-io/pipelines-orchestrate](https://github.com/gruntwork-io/pipelines-orchestrate)
+- [gruntwork-io/pipelines-execute](https://github.com/gruntwork-io/pipelines-execute)
+- [gruntwork-io/terragrunt-action](https://github.com/gruntwork-io/terragrunt-action)
+- [aws-actions/configure-aws-credentials](https://github.com/aws-actions/configure-aws-credentials)
+- [Codex-/return-dispatch](https://github.com/Codex-/return-dispatch)
+- [Codex-/await-remote-run](https://github.com/Codex-/await-remote-run)
+- [dsaltares/fetch-gh-release-asset](https://github.com/dsaltares/fetch-gh-release-asset)
+- [peter-evans/create-pull-request](https://github.com/peter-evans/create-pull-request)
```

Comment:
Since we recommend a wildcard allow for these GitHub Actions, we are putting a significant amount of security trust in these third parties. If Peter Evans updates his GitHub Action, 100% of our customers will get it, but we will have never reviewed it. Actually, the very fact that we publish this list means we're telling an attacker exactly which dependencies to compromise.

In traditional app dev, we can mitigate that risk with CVE checks and vulnerability scans. Does anything like that exist for GitHub Actions?

As far as next steps here, I've captured this in [CORE-1326](https://gruntwork.atlassian.net/browse/CORE-1326).

Reply:
>  If Peter Evans updates his GitHub Action, 100% of our customers will get it, but we will have never reviewed it

We're pinning the versions of each of the actions we use _inside_ of our actions and workflows. So we will _have_ to review any change to any 3rd party action, increment the version in our actions or workflows, then roll that out to customers.

Reply:
> We're pinning the versions of each of the actions we use inside of our actions and workflows.

Ah, that's a good point. 

> All that being said, ideally using these other actions is temporary while we put more logic into the binary.

Agreed!